content/04.results.md

```diff
@@ -6,7 +6,9 @@ Results section stub.
 
 This section will introduce the dataset (e.g., the histologies represented and what data types are included; Figure {@fig:openpbta-overview}A-B) and the process for contributing analytical code and to the manuscript (Figure {@fig:openpbta-overview}C-D).
 
-![An overview of the OpenPBTA project](images/figure_sketches/fig1-proposed-sketch.png){#fig:openpbta-overview}
+![An overview of the OpenPBTA project.
+A) The distribution of samples across short histologies and integrated diagnoses.
```

Comment:
I think this could use a bit more detail at least while we are writing this up. When you say `samples` what do you mean? Is it biospecimens? Is it `sample_id`? Participants?